app/api/crud.py

```diff
@@ -67,6 +69,10 @@ async def get(
         Non-imaging assessment completed by subjects.
     image_modal : str
         Imaging modality of subject scans.
+    pipeline_name : str
+        Name of pipeline run on subject scans.
```

Comment:
```suggestion
        Name of pipeline run on subject imaging data.
```

Reply:
In the interest of keeping things in line with what's in the node API I think we should stick with scans.